lldb/source/Expression/Materializer.cpp

```diff
@@ -483,19 +483,18 @@ class EntityVariableBase : public Materializer::Entity {
     }
 
     if (m_is_reference) {
-      DataExtractor valobj_extractor;
-      Status extract_error;
-      valobj_sp->GetData(valobj_extractor, extract_error);
+      auto valobj_extractor_or_err = valobj_sp->GetData();
 
-      if (!extract_error.Success()) {
+      if (auto error = valobj_extractor_or_err.takeError()) {
         err = Status::FromErrorStringWithFormat(
             "couldn't read contents of reference variable %s: %s",
-            GetName().AsCString(), extract_error.AsCString());
+            GetName().AsCString(), llvm::toString(std::move(error)).c_str());
         return;
```

Comment:
shouldn't we just do this from now on?

```suggestion
        return Status::FromErrorStringWithFormat(
            "couldn't read contents of reference variable %s: %s",
            GetName().AsCString(), extract_error.AsCString());
            GetName().AsCString(), llvm::toString(std::move(error)).c_str());
```

Reply:
Eventually, yes. You just have to draw the line somewhere for each commit.